frontend/lib/src/theme/globalStyles.ts

```diff
@@ -125,7 +125,7 @@ export const globalStyles = (theme: EmotionTheme): SerializedStyles => css`
   h1 {
     font-family: ${theme.genericFonts.headingFont};
     font-weight: ${theme.fontWeights.extrabold};
-
+    font-size: 2.75rem;
```

Comment:
this is just temporary until we clean up the headings `h*` from the globalStyles, right?

Reply:
correct, I think its better to handle this in a dedicated PR for cleaning up heading styles